packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -46,6 +46,28 @@ async function setup(): Promise<Fixture<Table<SimpleTableRecord>>> {
     );
 }
 
+async function sizeTableToGivenRowWidth(
```

Comment:
Given the other discussion about putting query selectors in the page object, do you think it would make sense for this to be a function on the page object rather than a function within this test file?